definitions/ext-router/definition.yml

```diff
@@ -11,9 +11,6 @@ synthesis:
   - attribute: provider
     value: kentik-router
 
-  tags:
-    src_addr:
```

Comment:
Maybe we should remove it from golden tags as well?

Reply:
yes! thanks!